test/foundry_tests/SDRewardsManager.t.sol

```diff
@@ -0,0 +1,271 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.16;
+
+import "../../contracts/library/UtilLib.sol";
+
+import "../../contracts/SDRewardManager.sol";
+import "../../contracts/StaderConfig.sol";
+import "../../contracts/SocializingPool.sol";
+
+import "../mocks/StaderTokenMock.sol";
+
+import "forge-std/Test.sol";
+import "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+import "@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol";
+import { IERC20Upgradeable } from "@openzeppelin/contracts-upgradeable/token/ERC20/IERC20Upgradeable.sol";
+
+contract SDRewardManagerTest is Test {
```

Comment:
typo : `SDRewardsManagerTest` 
missing `s`

Reply:
[49d6513](https://github.com/stader-labs/ethx/pull/253/commits/49d65130aaa1e19d52b13ddedc13e0a88212a0b8) given we have a single kind of reward, hence refactoring it to `SDRewardManager`